app/konnect/mesh-manager/service-mesh.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Configure a Mesh Global Control Plane with the Kubernetes demo app
+content_type: tutorial
+beta: true
+---
+
+Using Mesh Manager, you can create {{site.mesh_product_name}} global control planes to manage your {{site.konnect_saas}} mesh services. This guide explains how to configure a global control plane with {{site.mesh_product_name}} and the Kubernetes demo app. You will use Kubernetes to install various service mesh components.
+
+{:.important}
```

Comment:
I don't think this needs to be an `important` highlight. I think it can just be part of the paragraph. 
```suggestion

```